notify/ringcentral/ringcentral.go

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2022, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package ringcentral is the ringcentral notification package.
+package ringcentral
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/megaease/easeprobe/global"
+	"github.com/megaease/easeprobe/notify/base"
+	"github.com/megaease/easeprobe/report"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type NotifyConfig struct {
+	base.DefaultNotify `yaml:",inline"`
+	WebhookURL         string `yaml:"webhook" json:"webhook" jsonschema:"required,format=uri,title=Webhook URL,description=The RingCentral webhook URL"`
+}
+
+// Config configures the ringcentral notification
+func (c *NotifyConfig) Config(gConf global.NotifySettings) error {
+	c.NotifyKind = "ringcentral"
+	c.NotifyFormat = report.Text
+	c.NotifySendFunc = c.SendRingCentral
+	c.DefaultNotify.Config(gConf)
+	log.Debugf("Notification [%s] - [%s] configuration: %+v", c.NotifyKind, c.NotifyName, c)
+	return nil
+}
+
+func (c *NotifyConfig) SendRingCentral(title, msg string) error {
```

Comment:
Please fix this lint warning.

Reply:
Add comments for the exported method and type. Please help review.

Thanks.